column-profiling/common/src/main/java/com/r3/conclave/sample/dataanalysis/common/UserProfile.java

```diff
@@ -0,0 +1,32 @@
+package com.r3.conclave.sample.dataanalysis.common;
+
+public class UserProfile {
+    private String name;
+    private Integer age;
+    private String country;
+    private String gender;
```

Comment:
Best practice to make fields in a data class final

Reply:
Updated.